omigami/feature_selector.py

```diff
@@ -65,8 +130,39 @@ def fit(
         groups: NumpyArray = None,
         executor: Executor = None,
     ) -> FeatureSelector:
+        """
+        Implements the double CV feature selection algorithm. The method returns
+        the same FeatureSelector. If the samples are correlated, the `group` vector
+        can be used to encode arbitrary domain specific stratifications of the
+        samples as integers (e.g. patient_id, year of collection, etc.). If group
+        is not provided the samples are assumed to be i. i. d. variables.
+        To parallelize the CV repetition, an `executor` can be provided to split
+        the computation across processes or cluster nodes. So far, `loky` (joblib),
+        `dask`, and `concurrent` Executors are supported.
```

Comment:
Hm maybe we should say they are tested? - Are they?

Reply:
yes they are